### ISSUE TITLE
Adding NPM dependencies appears to break client scripts in production

### ISSUE THREAD

**Issue by sohara** [amannn/next-client-script#18]
Hi there,

I have run into an issue where it appears that importing anything from an npm package in a client script breaks the client script (it doesn't run on the page).

I have a repro case here: https://github.com/amannn/next-client-script/compare/master...sohara:test-npm-dependencies
(console logging works if you comment out the import)

Any ideas on why this is not working?

**Comment by amannn** [comment]
Hi @sohara and thanks for the report,

hmm, no – that's the first time I hear this. Isn't this what I'm doing here as well?

https://github.com/amannn/next-client-script/blob/de5ed6ffc9284e436764da076396515f13e859fe/packages/example/src/pages/index.client.tsx#L1

Would you be interested in looking into this and potentially providing a fix?

**Comment by sohara** [comment]
@amannn Yes, I suppose it is similar to that example (an npm dependency). However using other npm dependencies doesn't seem to work. I wasn't able to fix this issue and decided to use a parallel rollup build instead, with the bundles written to `/public`.

**Comment by amannn** [comment]
Hmm, interesting. Thanks for the report though. I'm currently not affected by this issue, so I'll leave it open for now in case someone else wants to look into it.
